rendering_styles/default.render.xml

```diff
@@ -11770,6 +11770,7 @@
 			<case tag="man_made" value="dyke"/>
 			<case engine_v1="true" tag="man_made" value="embankment"/>
 			<case tag="embankment" value="yes"/>
+            <case tag="cutting" value="yes"/>
```

Comment:
This is not correct. Our rendering engines can't correctly render cutting at that moment.